docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

```diff
@@ -1,10 +1,450 @@
+                        "value" : "72h"
+                     },
+                     "paths" : {
+                        "type" : "text",
+                        "value" : [
+                           "/var/log/nginx/error.log*"
+                        ]
+                     },
+                     "preserve_original_event" : {
+                        "type" : "bool",
+                        "value" : false
+                     },
+                     "processors" : {
+                        "type" : "yaml"
+                     },
+                     "tags" : {
+                        "type" : "text",
+                        "value" : [
+                           "test"
+                        ]
+                     }
+                  }
+               }
+            ],
+            "type" : "logfile"
+         },
+         ...
+         {
+            "enabled" : true,
+            "policy_template" : "nginx",
+            "streams" : [
+               {
+                  "compiled_stream" : {
+                     "hosts" : [
+                        "http://127.0.0.1:80"
+                     ],
+                     "metricsets" : [
+                        "stubstatus"
+                     ],
+                     "period" : "10s",
+                     "server_status_path" : "/nginx_status"
+                  },
+                  "data_stream" : {
+                     "dataset" : "nginx.stubstatus",
+                     "type" : "metrics"
+                  },
+                  "enabled" : true,
+                  "id" : "nginx/metrics-nginx.stubstatus-92f33e57-3165-4dcd-a1d5-f01c8ffdcbcd",
+                  "release" : "ga",
+                  "vars" : {
+                     "period" : {
+                        "type" : "text",
+                        "value" : "10s"
+                     },
+                     "server_status_path" : {
+                        "type" : "text",
+                        "value" : "/nginx_status"
+                     }
+                  }
+               }
+            ],
+            "type" : "nginx/metrics",
+            "vars" : {
+               "hosts" : {
+                  "type" : "text",
+                  "value" : [
+                     "http://127.0.0.1:80"
+                  ]
+               }
+            }
+         }
+      ],
+      "name" : "nginx-demo-123",
+      "namespace" : "default",
+      "package" : {
+         "name" : "nginx",
+         "title" : "Nginx",
+         "version" : "1.5.0"
+      },
+      "policy_id" : "d625b2e0-4c21-11ed-9426-31f0877749b7",
+      "revision" : 1,
+      "updated_at" : "2022-10-15T00:41:28.594Z",
+      "updated_by" : "1282607447",
+      "version" : "WzI5OTAsMV0="
+   }
+}
+----
+
+
+[discrete]
+[[get-enrollment-token-api]]
+== Get enrollment tokens
+
+To get a list of valid enrollment tokens from {fleet}, call
+`GET /api/fleet/enrollment_api_keys`. 
+
+This cURL example returns list of enrollment tokens.
```

Comment:
```suggestion
This cURL example returns a list of enrollment tokens.
```

Reply:
Great to have another pair of eyes on this. :-)